mobile-app/app/api/persistence/portfolio_currency_storage.ts

```diff
@@ -0,0 +1,19 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { PortfolioButtonGroupTabKey } from '@screens/AppNavigator/screens/Balances/components/TotalPortfolio'
+
+const KEY = 'WALLET.PORTFOLIO_CURRENCY'
+
+async function set (portfolioCurrency: PortfolioButtonGroupTabKey): Promise<void> {
+    await AsyncStorage.setItem(KEY, JSON.stringify(portfolioCurrency))
+}
+
+async function get (): Promise<PortfolioButtonGroupTabKey> {
+    // PortfolioButtonGroupTabKey.USD = 'USD'
+    const val = await AsyncStorage.getItem(KEY) ?? 'USD'
+    return JSON.parse(val)
```

Comment:
`JSON.parse('USD')` will throw an error, you'd see the error if you use incognito browser (nothing stored in local storage yet), 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/13292662/162660047-c5095aa7-d379-4dcc-9605-183d974266bd.png">

Reply:
using JSON.stringify will resolve this issue - selfnote :>